src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -573,6 +580,8 @@ else if (event instanceof GenericVoiceChannelUpdateEvent)
             onGenericVoiceChannelUpdate((GenericVoiceChannelUpdateEvent) event);
         else if (event instanceof GenericGuildUpdateEvent)
             onGenericGuildUpdate((GenericGuildUpdateEvent) event);
+        else if (event instanceof GenericGuildMemberUpdateEvent)
+            onGenericGuildMemberUpdateEvent(((GenericGuildMemberUpdateEvent) event));
```

Comment:
Remove the extra, unneeded enclosing parentheses.

Reply:
Didn't notice those, thanks